cs/test/core/Util.cs

```diff
@@ -271,6 +273,22 @@ public static ArraySegment<byte> SerializeUnsafeCB<T>(T obj)
             return output.Data;
         }
 
+        public static ArraySegment<byte> SerializeUnsafeCBWithPtr<T>(T obj)
```

Comment:
When you see a test case using `prtRoundtrip` you *know* you are testing with unmanaged memory because the type enforces that. If you see `memoryRoundtrip` than you are not sure until you look at the implementation of the Serialize/Deserialize delegate.